### DISCUSSION TITLE
Does the Google Chat Contact Point Work for Google Spaces?

### DISCUSSION THREAD

**Discussion by nategoethel** [grafana/grafana#68752] [Alerting]
Google Spaces are grouped under the [Google Chat product](https://workspace.google.com/products/chat/). Can this contact point also be used to send a message to a Space, as opposed to a group chat?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
